dask_sql/physical/rel/custom/create_table.py

```diff
@@ -62,11 +62,17 @@ def convert(
         except KeyError:
             raise AttributeError("Parameters must include a 'location' parameter.")
 
+        try:
+            gpu = kwargs.pop("gpu")
+        except KeyError:
+            gpu = False
+
```

Comment:
The `pop` function allows for a default argument, so you can write
```suggestion
        gpu = kwargs.pop("gpu", False)
```